diff --git a/dist/npm-CNmNicp2.js b/dist/npm-CNmNicp2.js
index 6d6e8df4910182c620d5b8727cc387efba64f535..4cf1b07517f1a218342ee2fc0ce656ded67ec29a 100644
--- a/dist/npm-CNmNicp2.js
+++ b/dist/npm-CNmNicp2.js
@@ -1298,7 +1298,7 @@ async function publish(client, pkg, distTag) {
 		if (packResult.exitCode !== 0) throw execFailure(`Failed to pack ${pkg.name}@${pkg.version}.`, packResult);
 		const publishArgs = ["publish", tarballPath];
 		if (distTag) publishArgs.push("--tag", distTag);
-		const publishResult = await x("npm", publishArgs, { nodeOptions: {
+		const publishResult = await x("bun", publishArgs, { nodeOptions: {
 			cwd: pkg.path,
 			...!isCI() && process.stdin.isTTY ? { stdio: "inherit" } : {}
 		} });
