diff --git a/dist/npm-e_qyiAtm.js b/dist/npm-e_qyiAtm.js
index c460719fcbbc9ebd5750a4d6aeccb561007e19ab..4ecd9cf59df8a0bc81170ba213fb61f3d2d50dea 100644
--- a/dist/npm-e_qyiAtm.js
+++ b/dist/npm-e_qyiAtm.js
@@ -1298,7 +1298,7 @@ async function publish(client, pkg, distTag) {
 		if (packResult.exitCode !== 0) throw execFailure(`Failed to pack ${pkg.name}@${pkg.version}.`, packResult);
 		const publishArgs = ["publish", tarballPath];
 		if (distTag) publishArgs.push("--tag", distTag);
-		const publishResult = await x("npm", publishArgs, { nodeOptions: {
+		const publishResult = await x("bun", publishArgs, { nodeOptions: {
 			cwd: pkg.path,
 			...!isCI() && process.stdin.isTTY ? { stdio: "inherit" } : {}
 		} });
