diff --git a/dist/npm-BrMZZcQU.js b/dist/npm-BrMZZcQU.js
index 90fafdb214167840da88b64c3f3a5ecd0f26df4c..02051cf82e191a7016adce5541bcc45203971249 100644
--- a/dist/npm-BrMZZcQU.js
+++ b/dist/npm-BrMZZcQU.js
@@ -1323,7 +1323,7 @@ async function publish(client, pkg, distTag) {
 		if (packResult.exitCode !== 0) throw execFailure(`Failed to pack ${pkg.name}@${pkg.version}.`, packResult);
 		const publishArgs = ["publish", tarballPath];
 		if (distTag) publishArgs.push("--tag", distTag);
-		const publishResult = await x("npm", publishArgs, { nodeOptions: {
+		const publishResult = await x("bun", publishArgs, { nodeOptions: {
 			cwd: pkg.path,
 			...!isCI() && process.stdin.isTTY ? { stdio: "inherit" } : {}
 		} });
