diff --git a/dist/npm-BIHQ7qby.js b/dist/npm-BIHQ7qby.js
index 4310df1d0f41e01ab2a0397d1c95a22d8e80d7cc..50baff1a3ee035b878f0f9551bc1906216f95fb3 100644
--- a/dist/npm-BIHQ7qby.js
+++ b/dist/npm-BIHQ7qby.js
@@ -1323,7 +1323,7 @@ async function publish(client, pkg, distTag) {
 		if (packResult.exitCode !== 0) throw execFailure(`Failed to pack ${pkg.name}@${pkg.version}.`, packResult);
 		const publishArgs = ["publish", tarballPath];
 		if (distTag) publishArgs.push("--tag", distTag);
-		const publishResult = await x("npm", publishArgs, { nodeOptions: {
+		const publishResult = await x("bun", publishArgs, { nodeOptions: {
 			cwd: pkg.path,
 			...!isCI() && process.stdin.isTTY ? { stdio: "inherit" } : {}
 		} });
